observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc

```diff
@@ -6,9 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: The {coo-full}
-include::snippets/technology-preview.adoc[leveloffset=+2]
-
+[IMPORTANT]
+====
+Until the approaching General Availability (GA) release of the Cluster Observability Operator (COO), which is currently in link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] (TP), Red Hat provides support to customers who are using Logging 6.0 or later with the COO for the logging UI plugin on {product-title} 4.16 or later. This support exception is temporary as the COO includes several independent features, some of which are still TP features, but the logging UI plugin is ready for GA.
```

Comment:
Inconsistent use of logging and Logging in files changed.

Reply:
Yes, it was not my choice, it is inconsistent between the docs.